### PR TITLE
Data model for the user status collection

### PR DESCRIPTION
**Pull request by heyrandhir**
- [x] documents the data model for user status

### CODE REVIEW

File: users-status/README.md [hunk 1]
```diff
@@ -0,0 +1,26 @@
+# Firestore user status collection data model
+
+```
+{
+  "id": "String",
+  "userId": "String",
+  "currentStatus": {
+    "state": "OOO | IDLE | ACTIVE",
+    "updatedAt": "TimeStamp",
+    "from": "TimeStamp ",
```
Comment (manish591) [inline users-status/README.md:10]: Can you remove the trailing space here

File: users-status/README.md [hunk 2]
```diff
@@ -0,0 +1,26 @@
...
+    "until": "TimeStamp",
+    "message": "String"
+  },
+  "futureStatus": {
+    "state": "OOO | IDLE | ACTIVE",
+    "updatedAt": "TimeStamp",
+    "from": "TimeStamp ",
```
Comment (manish591) [inline users-status/README.md:17]: Also here.

**Review by manish591 — APPROVED** [review]

**Review by sahsisunny — APPROVED** [review]

File: users-status/README.md [hunk 3]
```diff
@@ -0,0 +1,26 @@
...
+    "until": "TimeStamp",
+    "message": "String"
+  },
+  "futureStatus": {
+    "state": "OOO | IDLE | ACTIVE",
+    "updatedAt": "TimeStamp",
+    "from": "TimeStamp",
+    "until": "TimeStamp",
+    "message": "String"
+  },
```
Comment (ankushdharkar) [inline users-status/README.md:20]: Can this be 

```
status: {
  current:
  future:
}
```

**Review by ankushdharkar — CHANGES_REQUESTED** [review]
https://github.com/Real-Dev-Squad/website-data-models/pull/25/files#r1151269301

File: users-status/README.md (hunk 3 above)
Comment (heyrandhir) [reply users-status/README.md:20]: Created the refactor ticket [here](https://github.com/Real-Dev-Squad/website-backend/issues/975)

**Comment by heyrandhir** [comment]
> https://github.com/Real-Dev-Squad/website-data-models/pull/25/files#r1151269301

created the refactor ticket [here](https://github.com/Real-Dev-Squad/website-backend/issues/975)

**Pull request merged**